faker/providers/job/zh_CN/__init__.py

```diff
@@ -0,0 +1,483 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from .. import Provider as BaseProvider
+
+
+# Taiwan jobs from
+# http://www.104.com.tw/public/function01/utf8/jsonJobCat.js
+# on Fri May 20 23:13:46 CST 2016
```

Comment:
Is there any official source of data for zh_CN that we can use, instead of translating from zh_TW?